scripts/mklfpk.sh

```diff
@@ -19,17 +19,15 @@
 MKLFPK_URL_ROOT="https://github.com/intel/daal/releases/download/Dependencies/"
 MKLFPK_VERSION="20180112_10"
 MKLGPUFPK_VERSION="20191109"
-MKLFPK_ARCH=32e
+WITH_GPU=false
```

Comment:
We position oneDAL as primary thing in this repo. So i suggest to change this to default true

Reply:
np.
done